interp/tests/control/invoke/invoke-memory.futil.data.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Should this be committed here? I thought `fud` generates this for you.

Reply:
It shouldn't, but none of the python and fud machinery is installed for the CI interpreter test path and I didn't feel like messing around with that. So I did this. 

I should probably fix this

Reply:
Yes please. Hopefully copying the `fud` installation b/w workflows works. Github also has this interesting "composite" action thing that allows sharing stages between workflows: https://github.com/actions/runner/issues/646#issuecomment-901336347